sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -291,6 +293,22 @@ class ProjectOverviewView extends VerticalLayout implements HasHotbar, HasTitle
 
         projectGrid.setHeightMode(HeightMode.ROW)
 
+        HeaderRow headerRow = projectGrid.getDefaultHeaderRow()
+        headerRow.getCell("Subscription").setStyleName("header-with-tooltip")
+        headerRow.getCell("Subscription").setDescription("Select a project you want to get notified via email and get daily status updates.")
```

Comment:
```suggestion
        headerRow.getCell("Subscription").setDescription("Select a project to get status updates per email.")
```

Reply:
Do you think we should mention here that emails are only sent when specific statuses change?

Reply:
i think `updates` implies that imo

Reply:
I would still advise here to try to keep the sentence shorter and only include as much information as needed but enough to understand. I think the information on the daily updates is not necessary.